CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ if (ENABLE_CUDA)
     message(STATUS "CUDA compute architecture set to RAJA default sm_35 since it was not specified")
     set(CUDA_ARCH "sm_35" CACHE STRING "Set CUDA_ARCH to RAJA minimum supported" FORCE)
   endif()
```

Comment:
Note that this change is here (before BLT is loaded and outside of our `cmake/SetupCompilers.cmake` file because the flag needs to be set _before_ `enable_language(CUDA)` is called.